docs/product/dev-toolbar/faq.mdx

```diff
@@ -4,6 +4,55 @@ sidebar_order: 30
 description: "Frequently asked questions about the Dev Toolbar."
 ---
 
+<Expandable title="In what environments should I enable the Dev Toolbar?">
```

Comment:
I had suggested to name the npm package `developer toolbar` (I don't have strong feels about it), but if "Dev Toolbar" is the official product name for it, we should make the npm package name match.

(why can't this be "Sentry Toolbar", does it need dev at all?)

Reply:
renaming the thing makes sense. the repo is called `sentry-toolbar` already.

I think using `development`, especially in the npm package name, was meant to make it not sound like an environment or a dev/alpha/beta tag. A proper rename avoids any confusion in that area.

Reply:
Put up a rename PR: https://github.com/getsentry/sentry-toolbar/pull/223

The current state is that the npm publish of `@sentry/developer-toolbar` (the old name) isn't complete yet, there's a bug in the pipeline.

I've asked for help with that pipeline bug, so if it's not disruptive to debugging we can land the rename name. I'll check in on that. Otherwise I might just keep working at that pipeline problem first which means we will publish the old name once. After that's working we can land the rename.